src/screens/Asset/ReceiveModal.js

```diff
@@ -342,6 +354,7 @@ const InfoView = styled.View`
   align-items: center;
   justify-content: space-between;
   width: 100%;
+  max-height: ${Dimensions.get('screen').height * 0.75}px;
```

Comment:
FWIW we have a function that gets the device width in `common.js` at line 445 - `getDeviceWidth()`.